### PR TITLE
Added the native species Lembala'Lam.

### PR DESCRIPTION
**Pull request by Deepsloth**
Cleaned, fixed, tweaked and with comments.

### CODE REVIEW

**Comment by Deepsloth** [comment]
Successor of https://github.com/freeorion/freeorion/pull/816

File: default/scripting/species/SP_LEMBALALAM.focs.txt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            priority = 130
+            effects = [
+                SetTargetPopulation value = 5 accountinglabel = "IMMORTAL_LABEL"
+            ]
+
+        // Give techs to empire
+        EffectsGroup
+            scope = Source
+            activation = And [
+                NotOwnerHasTech name = "SPY_STEALTH_1"
```
Comment (geoffthemedio) [inline default/scripting/species/SP_LEMBALALAM.focs.txt]: Should probably be a space between Not and Owner

File: default/scripting/species/SP_LEMBALALAM.focs.txt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                        tag = "species" data = "SP_LEMBALALAM"
+                        tag = "tech" data = "SPY_STEALTH_1"
+                    ]
+                    empire = Source.Owner
+           ]
+
+        EffectsGroup
+            scope = Source
+            activation = And [
+                NotOwnerHasTech name = "SPY_STEALTH_2"
```
Comment (geoffthemedio) [inline default/scripting/species/SP_LEMBALALAM.focs.txt]: ...

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -5129,6 +5129,11 @@ On %planet%: Ancient Ruins were excavated, but only tantalizing hints of ancient
 EFFECT_ANCIENT_RUINS_EMPTY_LABEL
 Ancient Ruins Empty Excavation
 
+EFFECT_NATIVES_TECH
+The technology %tech% was learned from the %species%!
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Mention the planet also?

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -11440,6 +11462,9 @@ AVERAGE_POPULATION
 GOOD_POPULATION
 +	Good Population 125%
 
+FIXED_LOW_POPULATION
+−−	Population is fixed at 5
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: population or target population?

if the actual population should also be fixed, you need an extra effect to set it as such...

**Comment by Deepsloth** [comment]
I want to resolve the conflicts in en.txt.
But how can i resolve the conflict without merging with master?

**Comment by Morlic-fo** [comment]
Might be a better way of doing it but this should work:

1, checkout this_branch
2. rebase master
3. When prompted, resolve conflicts in files
4, git add -a
5. rebase --continue

Repeat 3-5 as often as needed

**Comment by Vezzra** [comment]
Closing this as replacement PR #1016 has been posted.

**Pull request closed without merging**